smarts/sstudio/genscenario.py

```diff
@@ -153,7 +210,7 @@ def gen_social_agent_missions(
     seed: int = 42,
     overwrite: bool = False,
 ):
```

Comment:
Shouldn't this be removed if it's deprecated? Or do we want to warn the user to use `gen_social_agents` instead?

Reply:
I do not want to immediately remove this method. I want to give a chance for users to move to the `gen_social_agents(...)` method first.